packages/input/src/index.vue

```diff
@@ -253,8 +256,8 @@ export default defineComponent({
       resize: props.resize,
     }))
     const inputDisabled = computed(() => props.disabled || elForm.disabled)
+    const upperLimit = computed(() => props.maxlength)
```

Comment:
Since we have `props.maxlength` why bother creating a computed which only returns `props.maxlength` anyway?

Reply:
Got it.